lib/steep/errors.rb

```diff
@@ -127,7 +138,7 @@ def initialize(node:, type:, method:)
       end
 
       def to_s
-        "#{location_to_str}: NoMethodError: type=#{type}, method=#{method}"
+        format_message "type=#{type}, method=#{method}", class_name: "NoMethodError"
```

Comment:
[note] Only this class name is different from one in the message (`NoMethod` ≠ `NoMethodError`):

https://github.com/soutaro/steep/blob/6eec2fbd6ce85b498579f18e3df8aee171146a03/lib/steep/errors.rb#L119